### ISSUE TITLE
Missing informations about available variables inside a "looping" tags

### ISSUE THREAD

**Issue by dadaxr** [statamic/docs#808]
Some variable (like "count" / "index" / "last" / "first" / "prev" / "next" etc...) are automatically added and updated inside loopish tags (ie collection / loop / foreach / replicator fields ... )

For now, those variables are mentioned in the documentation for some tags only : 
ie : for "collection" tag it's mentioned  https://statamic.dev/tags/collection#variables) 
but not for "loop" tag : https://statamic.dev/tags/loop 
neither for "foreach" tag, and neither for tag working in a "loopish" way (like "assets" or replicator fields for instance).

Instead of replicating available varibles related to loop in each of these tag docs maybe it could be wise to add a dedicated page explaining that particular behavior and available variables, and reference that page inside the different tag concerned.

What do you think ?

**Comment by Dr-Chronosphere** [comment]
Yes, I ran into this with the nav tag, and was happy to see count and index were available as variables, but was banging my head against the wall trying to find where this behavior was ever documented!
